### ISSUE TITLE
New sensor wirelessDoublePIRDetector

### ISSUE THREAD

**Issue by wifi75** [petrleocompel/hikaxpro_hacs#114]
see log

[home-assistant_hikvision_axpro_2024-05-24T09-05-59.140Z.log](https://github.com/petrleocompel/hikaxpro_hacs/files/15430955/home-assistant_hikvision_axpro_2024-05-24T09-05-59.140Z.log)

**Comment by petrleocompel** [comment]
```
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 575, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/__init__.py", line 107, in async_setup_entry
    await hass.async_add_executor_job(coordinator.init_device)
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/__init__.py", line 185, in init_device
    self.load_devices()
  File "/config/custom_components/hikvision_axpro/__init__.py", line 223, in load_devices
    devices = self._load_devices()
              ^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/__init__.py", line 236, in _load_devices
    return ZonesConf.from_dict(response.json())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/model.py", line 981, in from_dict
    list = from_list(ZoneConfListWrap.from_dict, obj.get("List"))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/model.py", line 29, in from_list
    return [f(y) for y in x]
            ^^^^
  File "/config/custom_components/hikvision_axpro/model.py", line 965, in from_dict
    zone = ZoneConfig.from_dict(obj.get("Zone"))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hikvision_axpro/model.py", line 715, in from_dict
    detector_type = DetectorType(obj.get("detectorType"))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 744, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1158, in __new__
    raise ve_exc
ValueError: 'wirelessDoublePIRDetector' is not a valid DetectorType
```

**Comment by petrleocompel** [comment]
Released in version [v2.1.2](https://github.com/petrleocompel/hikaxpro_hacs/releases/tag/v2.1.2)
